blog/2023-07-27-introducing-substratus.md

```diff
@@ -0,0 +1,137 @@
+---
+slug: introducing-substratus
+title: Introducing Substratus
+authors:
+  name: Brandon Bjelland
+  title: Co-founding Engineer, CXO
+  url: https://github.com/brandonjbjelland
+  image_url: https://avatars.githubusercontent.com/u/2502520?v=4
+tags: [hello world, introduction, oss launch]
+---
+# Announcing Substratus: A Multi-cloud Substrate for AI
+
+We're beyond excited to announce Substratus (s8s), an open-source platform to
+manage machine learning primitives in Kubernetes.
```

Comment:
I'm not a huge fan of the s8s naming. Let Nick be the judge on this one. If it's 2v1 then I'm fine with it.

Reply:
> We're beyond excited to announce Substratus (s8s), an open-source platform to
manage machine learning primitives in Kubernetes.

What you think about this instead?
"We're beyond excited to announce Substratus, an open-source platform to
Build, Train, and Serve state of the art Models on top of Kuberentes."

Reply:
I'm mostly just experimenting with s8s to see if it's sticky or not. Not opposed to removing it but I think esp early on we should experiment to see what if anything helps make the project easy to remember.

Reply:
I like your suggested replacement - I think management of ML primitives is mentioned elsewhere anyway.

Reply:
Hmm that's a good point, s8s might stick better than substratus. Lets keep it for now.